server/chat-plugins/poll.ts

```diff
@@ -306,7 +309,15 @@ export const commands: ChatCommands = {
 			if (new Set(options).size !== options.length) {
 				return this.errorReply("There are duplicate options in the poll.");
 			}
-
+			// if there's a poll in place, queue instead.
+			if (cmd === 'queue' && !room.queuedActivity && room.minorActivity) {
+				room.queuedActivity = new Poll(room, {source: params[0], supportHTML}, options, multi);
+				this.modlog('QUEUEPOLL');
+				return this.privateModAction(`${user.name} queued a poll.`);
+			}
```

Comment:
Also, what happens if you try to make a poll, but a poll (but no queued poll) already exists?

Reply:
should be properly blocked, unless /poll queue is used, now.

Reply:
Mia! I can't just tell you the answer every time! I'm trying to get you to understand things without needing them to be told to you.

Maybe you'll see what I mean if you actually try it on your code.

Reply:
(Although ideally one day you'll be able to figure out this sort of thing just by reading your code.)

Reply:
Oh, wait, nvm I'm dumb and was reading your code wrong, my bad.

Reply:
Sorry, i've been doing it for the most part - just made the above fix while not thinking entirely.

Reply:
Oh, lol.

Reply:
Was... very confused for a minute.